services/QuillLMS/app/services/google_integration/lesson_announcement.rb

```diff
@@ -1,119 +0,0 @@
-require 'google/api_client'
-
-module GoogleIntegration
-  class LessonAnnouncement
```

Comment:
I was going to comment that you didn't remove any references to `LessonAnnouncement`, but I just checked the codebase and there aren't any! So this is apparently vestigial code. Thanks for removing.